src/maker.rs

```diff
@@ -88,44 +82,38 @@ impl Maker {
         self.dai_balance = balance;
     }
 
-    pub fn track_failed_balance_update(&mut self, error: anyhow::Error) {}
-
-    fn rate_is_up_to_date(&self) -> bool {
-        let current_timestamp = Utc::now().timestamp();
-        let mid_market_rate_timestamp = self.mid_market_rate.timestamp.timestamp();
-        let acceptable_timestamp = (current_timestamp - self.max_rate_ticker_interval_secs);
-
-        mid_market_rate_timestamp >= acceptable_timestamp
+    pub fn track_failed_rate_update(&mut self) {
```

Comment:
:wrench: I think it would be clearer if the name of the function said what it does rather than why it would be called. You could even just inline it.

Reply:
I kindoff like the function better than just making the `mid_market_rate` public. This way it is a clearer interface I feel. 
I renamed it to `invalidate_rate()`.

Reply:
Makes sense :D I pushed https://github.com/coblox/nectar/pull/38/commits/a14ff830809521c4384db57ff6286a71fc110f35 to fix the typo in the name.